config/config.go

```diff
@@ -176,6 +176,8 @@ type Config struct {
 	// where M is the element literal length and w is the number of bytes required for the maximum-length character in the character set.
 	// See https://dev.mysql.com/doc/refman/8.0/en/string-type-syntax.html for more details.
 	EnableEnumLengthLimit bool `toml:"enable-enum-length-limit" json:"enable-enum-length-limit"`
```

Comment:
Is it seems that this config is not **ONLY** used for lvs?
enable-->enables

Reply:
1. yes, but that is a reason why user need it. 
2. enable --> enables: fixed.